src/models/parties/parties.js

```diff
@@ -6,40 +6,42 @@ export const DEFAULT_PARTY_COUNT = 5;
 export const MAX_PARTY_COUNT = 10;
 
 export function makeParties() {
-  let names = Set([
-    'asparagus',
-    'corn',
-    'cabbage',
-    'turnip',
-    'radish',
-    'carrot',
-    'onion',
-    'spinach',
-    'mushroom',
-    'eggplant'
-  ]);
+  let names = [
+    {'asparagus': 'rgb(151, 190,17)',
+      id: 1
+    },
+    {'corn': 'rgb(255,255,51)',
+      id: 2
+    },
+    {'cabbage': 'rgb(204,255,153)',
+      id: 3
+    },
+    {'turnip': 'rgb(204,0,204)',
+      id: 4
+    },
+    {'radish': 'rgb(255,0,127)',
+      id: 5
+    },
+    {'carrot':'rgb(255,108,0)',
+      id: 6
+    },
+    {'onion': 'rgb(223,223,172)',
+      id: 7
+    },
+    {'spinach': 'rgb(40,89,12)',
+      id: 8
+    },
+    {'mushroom': 'rgb(255,204,153)',
+      id: 9
+    },
+    {'eggplant':'rgb(102,0,102)',
+      id: 10
+    }
+  ];
+  let randomIdNumber = Math.floor((Math.random() * 10) + 1)
+  let render = names.map(veggie => {
+    
+  }
 
-  let colors = Set([
-    'rgb(18, 147, 154)',
-    'rgb(121, 199, 227)',
-    'rgb(26, 49, 119)',
-    'rgb(255, 152, 51)',
-    'rgb(239, 93, 40)'
-  ]);
-
-  return Map(
```

Comment:
We still need this `return` statement. Part of the reason that `Legend.js` was complaining was because, without anything returned from this function, when it tries to render the parties, it has no data.